### DISCUSSION TITLE
How to give user 'Click and Place' option?

### DISCUSSION THREAD

**Discussion by streetcodec** [google/model-viewer#4279] [Q&A]
I want to give user option to simply click on a button moving on wall (using ar wall placement) and the model should appear on the wall and stay there, no matter how the user moves. Is this possible with model viewer?

**Comment by elalish** [comment]
Our WebXR UX allows the user to drag the model across the wall instead, allowing it to be repositioned. However, this is only as good as ARCore's wall detection, which doesn't always work. Have you tried it? We have no way to customize anything about Quick Look (iOS) or Scene Viewer.

**Reply by streetcodec** [reply]
Yeah! I have tried it. In Android it is still working fine enough, but in iOS devices the model suddenly becomes humongous, as if a plane very near to the user is detected. Can we correct this? Moreover, can we have some way to just fix the model if user is satisfied? As what I have found during testing, moving the device, the object still moves considerably.

**Reply by elalish** [reply]
You'll have to file iOS QuickLook bugs on WebKit (@grorg) as we have no ability to change anything. Are you using our USDZ autogeneration feature, or your own `ios-src`? It sounds like there might be a problem with units somewhere.

**Reply by streetcodec** [reply]
I am simply using <model viewer src = "xyz.glb"> tag. I am kinda new to model viewer, so I am not well versed with many functionalities, but what I am trying to accomplish is showing models on wall via browsers, which is going well in android but not in ios devices. I have done nothing different for ios devices. Looks like I have to file iOS QuickLook bugs on WebKit, as you suggested!
